src/decoder/jpeg_decoder.cpp

```diff
@@ -53,5 +53,9 @@ string decode_jpeg_buffer(char * buffer, size_t size, CImg<unsigned char> ** cim
     jpeg_finish_decompress(&cinfo);
     jpeg_destroy_decompress(&cinfo);
 
+    // TODO: implement getting metadata from GIFs; this is a placeholder
```

Comment:
comment should this:
// TODO: implement getting metadata from JPEGs; this is a placeholder